metasv/main.py

```diff
@@ -321,9 +327,10 @@ def run_metasv(sample, reference, pindel_vcf=[], pindel_native=[], breakdancer_v
                                            min_contig_len=100, age_workdir=age_tmpdir)
 
         final_bed = os.path.join(workdir, "final.bed")
-        if ignored_bed:
-            pybedtools.BedTool(breakpoints_bed).cat(pybedtools.BedTool(ignored_bed), postmerge=False).sort().saveas(
-                final_bed)
+        if ignored_bed:  # Marghoob what?
```

Comment:
@marghoob What is this ignored BED thing? It seems to determine if "final.bed" or "breakpoints" will be used? This seems strange, can you explain the logic there? Thanks!